2021/svg.html

```diff
@@ -107,7 +107,7 @@ <h1 id="title">
         <a href="https://www.w3.org/2004/01/pp-impl/19480/join">Join the SVG Working Group.</a>
       </p>
       <p style="padding: 0.5ex; border: 1px solid green">
-        This proposed charter is available on <a href="https://w3c.github.io/charter-drafts/2021/svg.html">GitHub</a>.
+        This proposed charter is available on <a href="https://github.com/w3c/charter-drafts/blob/gh-pages/2021/svg.html">GitHub</a>.
```

Comment:
Is it usual to link to the HTML source instead of the rendered view? I personally don't mind, but I thought it might make the document easier to review.

Reply:
as far as I know, the link to github is meant to help people make pull request - linking to self doesn't really help that.

Reply:
Yeah, that makes sense... I'd imagined this draft would be copy/pasted somewhere as the review version.